Update makedepend.SH from perl-current to handle changed preprocessor
output from new gcc.

diff -Naur --exclude=debian perl-5.8.8.orig/makedepend.SH perl-5.8.8/makedepend.SH
--- perl-5.8.8.orig/makedepend.SH	2003-06-06 04:11:10.000000000 +1000
+++ perl-5.8.8/makedepend.SH	2007-09-16 10:26:25.000000000 +1000
@@ -128,7 +128,7 @@
     *.y) filebase=`basename $file .y` ;;
     esac
     case "$file" in
-    */*) finc="-I`echo $file | sed 's#/[^/]*$##`" ;;
+    */*) finc="-I`echo $file | sed 's#/[^/]*$##'`" ;;
     *)   finc= ;;
     esac
     $echo "Finding dependencies for $filebase$_o."
@@ -167,7 +167,9 @@
             -e '/^#.*<builtin>/d' \
             -e '/^#.*<built-in>/d' \
             -e '/^#.*<command line>/d' \
+            -e '/^#.*<command-line>/d' \
 	    -e '/^#.*"-"/d' \
+	    -e '/^#.*"\/.*\/"/d' \
 	    -e '/: file path prefix .* never used$/d' \
 	    -e 's#\.[0-9][0-9]*\.c#'"$file.c#" \
 	    -e 's/^[	 ]*#[	 ]*line/#/' \
